mkosi/resources/mkosi-initrd/mkosi.conf.d/10-serpentos.conf

```diff
@@ -0,0 +1,39 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+[Match]
+Distribution=serpentos
+
+[Content]
+# Completely override the default Packages
+Packages=
+Packages=
+        systemd
+        dash
+        # Various libraries that are dlopen'ed by systemd
+
+        # File system checkers for supported root file systems
+        e2fsprogs
+        xfsprogs
+
+        # fsck.btrfs is a dummy, checking is done in the kernel.
+
+        util-linux
```

Comment:
bash? less? `dash` is not a useful emergency login shell so we should definitely make sure `bash` is available.

Reply:
@hunger I'm open to having conversations about all of these decisions, but please understand that we're not going to special case SerpentOS. We want to provide a consistent experience across distributions. We're not going to make exceptions for just SerpentOS. So if you want to see this changed, please do so in separate PRs and do so for all supported distributions